devTools/executeTests.cmd

```diff
@@ -0,0 +1,55 @@
+echo off
+REM Execute all relevant tests
+
+REM ---------------------------------------------------------------------------
+cls
+
+REM ---------------------------------------------------------------------------
+REM default values
+SET WINCC_OA_INSTALL_PATH=C:\Siemens\Automation\WinCC_OA\
```

Comment:
The Install Path includes the version number by default, and one can also remove it during installation. So not including it here and instead maybe adding WINCC_OA_VERSION hardcoded afterwards somewhere might not be correct.

As a suggestion, read the installation dir by default from the registry. This is the default generic path:
RegistrySearchPath: "Software\\$(var.VendorShortName)\\$(var.ProductNameBlankless)\\$(var.VersionMajorMinor)\\"

The installdir should be listed there

Reply:
> The Install Path includes the version number by default, and one can also remove it during installation. So not including it here and instead maybe adding WINCC_OA_VERSION hardcoded afterwards somewhere might not be correct.
> 
> As a suggestion, read the installation dir by default from the registry. This is the default generic path: RegistrySearchPath: "Software\$(var.VendorShortName)\$(var.ProductNameBlankless)\$(var.VersionMajorMinor)\"
> 
> The installdir should be listed there

good idea, I will try that

Reply:
The problem with the install path still exists, normally install path is defined including version number, and the versionnumber is no requirement, it can be completely removed from the path.
This path setting here has another behaviour.

Reply:
I read the path from registry like see devTools/registerHelperProject.cmd

```
set REG_KEY=HKLM\Software\ETM\WinCC_OA\%WINCC_OA_VERSION%\

set WINCC_OA_INSTALL_PATH=

for /f "tokens=2,*" %%a in ('reg query %REG_KEY% /v INSTALLDIR ^| findstr INSTALLDIR') do (
    set WINCC_OA_INSTALL_PATH=%%b
)
```

What is wrong?

Reply:
Thank you for clarification, I looked at the old commit, sry for that, lgtm.